test/distributed/fsdp/test_fsdp_summon_full_params.py

```diff
@@ -38,9 +38,11 @@
 
 
 def _run_test_summon_full_param_writeback(cls, writeback, cpu_offload, modify_outer):
-    model = FSDP(
-        nn.Sequential(FSDP(nn.Linear(5, 5, bias=False)), nn.Linear(5, 3, bias=False))
-    ).cuda(cls.rank)
+    lin1 = FSDP(nn.Linear(5, 5, bias=False).cuda(cls.rank), cpu_offload=cpu_offload)
+    lin2 = nn.Linear(5, 3, bias=False).cuda(cls.rank)
+    model = FSDP(nn.Sequential(lin1, lin2), cpu_offload=cpu_offload)
+    if not cpu_offload:
+        model = model.cuda(cls.rank)
```

Comment:
why do we need this? I assume when `cpu_offload==False`, the model is already on cuda, no?

Reply:
Right, we can remove this because lin1, lin2 are already on CUDA device.